document/sections/03a-motivation.adoc

```diff
@@ -0,0 +1,96 @@
+ - data quality evaluation of the given 3D-data;
+ - Geometric and topological description in natural language.
+
+==== We aim to support Data Providers by:
+ - Maximize the use of 3D-geometry and 3D-topology data;
+ - Defining multiple 3D topology or geometries;
+ - To quality-assure 3D geospatial data.
+
+
+=== Semantic-functional value of 3D-GeoSPARQL
+ Value for data reasoning, formal description, and semantic interoperability
+
+==== Meaningful creation and querying of 3D geometry
+ Whether manually or programmatically defining 3D geometries in BIM, GEO or CG environments, the GeoSPARQL standard could provide or connect to vocabulary to represent 3D structures. A 3D cube, for example, can be represented in multiple ways:
+
+ - As a set of bounding points, lines, and faces;
+ - As a base surface with an extrude function;
+ - As an 3D geometric primitive (e.g., the concept of a 'cube' as a spatial figure).
+
+ When GeoSPARQL offers the means to define and link 3D geometries in these different ways, it allows both humans and machines to interpret and interact with the data effectively.
+
+==== Meaningful creation and querying of 3D topological relationships
+ Is “3D Geometry A” inside, touching, overlapping, intersecting or above “3D Geometry B”?
+
+ Being able to describe and use such topological relationships is crucial for conducting spatial analyses, formulating rules, and deriving knowledge from differten heterogeneous datasets.
+
+ Beyond describing topological relationships, one should be able to query them:
+
+ What is the spatial relationship between 3D Geometry A and 3D Geometry B?
+
+ Such queries should return the appropriate topological result (e.g., "intersects"), which is essential for use cases like clash detection in design validation
+
+==== Derivation of geometric and topological knowledge
+ From explicitly modeled geometric and topological data, one should be able to infer implicit knowledge.
+
+ For instance, if 3D Geometry A is 3D inside 3D Geometry B, then we can infer that:
+
+ A does not "touch" B, and
+ A does not "overlap" with B,
+
+ even if these facts are not explicitly stated.
+
+ For entailment we aim to be able to answer queries like: "Does this tree have a representation in 3D geometry?"
+
+ If the tree is defined using a MultiSolid geometry, then the answer should be "yes," even if this is not explicitly declared as such.
+
+==== Adding multiple 2D and 3D geometries to a single feature
+ GeoSPARQL supports linking multiple geometries (e.g., 2D and 2.5D) to a single feature. We aim to extend this with 3D geometries. 
+
+ For example, a tree in the real world may be represented in databases with both a 2D and a 3D geometry.
+
+ This should not require duplication of the tree entity — there can be one tree instance with both geometries linked, avoiding redundancy in the data model.
+
+=== Value for user experience, visualization, and application use of 3D-GeoSPARQL
```

Comment:
I can imagine everything under this heading being moved to the Benefits section. Perhaps replaced by one or two sentences in the Motivation giving some examples of expected benefits, because these benefits have motivated us of course.